crypto/x509/v3_ist.c

```diff
@@ -102,9 +102,6 @@ static int i2r_issuer_sign_tool(X509V3_EXT_METHOD *method,
         return 0;
     }
     if (ist->signTool != NULL) {
-        if (new_line == 1) {
-            BIO_write(out, "\n", 1);
-        }
```

Comment:
This is silly...

Reply:
You want the dead code put back in?
Why?

Reply:
The code is back and I've marked coverity as a false positive for this one.

Reply:
I figured it showed the pattern to follow, should anyone want to add another printed line before this one. But never mind, the code fragments are small enough to make it obvious.

Reply:
I've reverted this already, might as well leave it in.